Sources/WoosmapGeofencing/Monitor.swift

```diff
@@ -59,24 +59,36 @@ public protocol VisitServiceDelegate: AnyObject {
 }
 
 /// Location manage protocal extend  LocationManagerProtocol
-public protocol LocationManagerProtocol {
-    var desiredAccuracy: CLLocationAccuracy { get set }
-    var allowsBackgroundLocationUpdates: Bool { get set }
-    var distanceFilter: CLLocationDistance { get set }
-    var pausesLocationUpdatesAutomatically: Bool { get set }
-    var delegate: CLLocationManagerDelegate? { get set }
-    var monitoredRegions: Set<CLRegion> { get }
-    func requestAlwaysAuthorization()
-    func startUpdatingLocation()
-    func stopUpdatingLocation()
-    func startMonitoringSignificantLocationChanges()
-    func stopMonitoringSignificantLocationChanges()
-    func stopMonitoring(for: CLRegion)
-    func startMonitoring(for: CLRegion)
-    func startMonitoringVisits()
+
+public class LocationManagerProtocol: CLLocationManager {
+//    var desiredAccuracy: CLLocationAccuracy { get set }
```

Comment:
Why not just remove these comments?

Reply:
To avoid conflict between another pending PR : https://github.com/Woosmap/geofencing-core-ios-sdk/pull/23

I will remove it in the next version when I address issue :  https://github.com/Woosmap/geofencing-core-ios-sdk/issues/22